### ISSUE TITLE
kube-apiserver certificate changed is shown on every run

### ISSUE THREAD

**Issue by superseb** [rancher/rke#1405]
**RKE version:**
v0.2.4

**Steps to Reproduce:**
- Create one node and add it to `cluster.yml` with all roles
- Run `rke up`

**Results:**
`kube-apiserver certificate changed` is shown on every run (without any changes to the cluster)

```
INFO[0000] [certificates] kube-apiserver certificate changed, force deploying certs 
```

**Comment by jiaqiluo** [comment]
The bug is reproduced in rke version v0.2.6

Steps:
- create a cluster.yml file with one node with all roles
- rke up to build the cluster
- rke up again (no change in the cluster yml file )
- see the `kube-apiserver certificate changed` message
- rke up again (no change in the cluster yml file )
- see the `kube-apiserver certificate changed` message again

```
➜  rke up  --config rancher-cluster.yml
...
INFO[0000] [certificates] kube-apiserver certificate changed, force deploying certs
...
```

**Comment by jiaqiluo** [comment]
The bug is seen in  rke version v0.2.7, too

Wait for an RC to validate the fix.

**Comment by deniseschannon** [comment]
Available with v0.3.0-rc6

**Comment by jiaqiluo** [comment]
The bug fix is validated in v0.3.0-rc6

Following the same steps as my previous comment, do not see the `kube-apiserver certificate changed	` message anymore
